style.css

```diff
@@ -810,15 +810,15 @@ a {
     transition: transform 0.05s ease-out;
   }
 
-  #contact-container {
+  /* #contact-container {
     background-image: url(./images/contact_backward.png);
     background-size: cover;
     background-repeat: no-repeat;
     z-index: 2;
     margin: -120px auto;
     padding: 100px 20px 0;
     height: 112vh;
-  }
+  } */
 
```

Comment:
- [ ] This seems like a new development but as professional developers, we must always strive to keep our code clean and DRY. This comment section makes it slightly messy. If you are not using the styles again, I suggest you just remove them all the same since they're no longer needed 💪 .